src/main/java/entities/Feed.java

```diff
@@ -3,31 +3,20 @@
 import java.util.ArrayList;
 
 public class Feed {
-    private ArrayList<Post> posts;
+    private final ArrayList<Post> posts;
     private ArrayList<Post> displayedPosts;
 
+    /**
+     * Construct a Feed object.
+     *
+     * @param posts The list of posts that will be stored inside the feed object.
+     */
     public Feed(ArrayList<Post> posts) {
         this.posts = posts;
         this.displayedPosts = posts;
     }
 
-    public void setFilter(String filter) {
```

Comment:
Is there a reason for deleting these methods? Shouldn't we be able to set a filter on a Feed?